msdt-2/untitled5.py

```diff
@@ -0,0 +1,203 @@
+
+from scipy import signal
+from scipy.io import wavfile
+import sklearn.neural_network as nn
+import sklearn.model_selection as ms
+import sklearn.preprocessing as pp
+import sklearn.metrics as mt
+import matplotlib.pyplot as plt
+import os
+import numpy as np
+
+def write_to_csv(file_names,output_file,data_array):
+    output = open(output_file, 'w')
+    output.write(",")
+    for i in file:
+        output.write(i+',')
+    for i in range(67):
+        output.write('\n'+file_names[i])
+    for j in range(67):
+        output.write(','+str(data_array[i,j]))
+    output.close()
+
+def prepare_and_print(data_array,labels_array,metric_type):
+    data_array  = (data_array-np.min(data_array))/(np.max(data_array)-np.min(data_array))#normalize data
+    data_array_t = np.transpose(data_array)                                         #transpose to get a second array in a pair KL(x,y)
+    data_array =np.triu(data_array)                                                 #remove duplicate pairs
+    data_array_t =np.triu(data_array_t)                                   
+    data_array_label1 = conditions_label* data_array                       
+    data_array_label2 = data_array-data_array_label1
+    data_array_t_label1 = conditions_label* data_array_t
+    data_array_t_label2 = data_array_t-data_array_t_label1                          #get arrays of pairs for each label
+
+    plt.figure()
+    plt.xlabel(metric_type+'(a,b)')
+    plt.ylabel(metric_type+'(b,a)')
+    plt.scatter(data_array_label1, data_array_t_label1,s=0.5)
+    plt.scatter(data_array_label2, data_array_t_label2,s=0.5)
+    plt.show()
+
+def make_class_labels(file_names):
+    class_labels = []
+    for i in file_names:
+        label_tmp=0

```

Comment:
It's more better if are spaces between operators =, +, -, / and etc.
```suggestion
        label_tml = 0
```